docs/react-v9/contributing/api-extractor.md

```diff
@@ -0,0 +1,25 @@
+### What is API Extractor?
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31651&commitId=563df30d25b21c5504311fa6a1c5f61fafe17fae&env=prod&clientType=FLUENTUI)


  <details><summary><b>Card Converged - Interactive</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.normal.chromium.png|0| Added|

</details>


  <details><summary><b>Card Converged - Selectable</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Filled - RTL.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.normal.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.selected.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.normal.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.selected.chromium.png|0| Removed|

</details>


  <details><summary><b>Image Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image Converged.Image Variations Fallback.normal.chromium.png|0| Added|

</details>


  <details><summary><b>SwatchPicker Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.default.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>